tests/fixtures/major-name-modification.json

```diff
@@ -27,7 +28,8 @@
         "type": "Feature",
         "properties": {
           "name": "Bangaluru",
-          "wikidata": "Q40435"
+          "wikidata": "Q40435",
+          "osm:version": 9
```

Comment:
@geohacker Should the version be > 10 to make sure we check just the name modification and version wise things are good here: https://github.com/mapbox/osm-compare/blob/master/comparators/major-name-modification.js#L21